include/uv/win.h

```diff
@@ -550,7 +550,10 @@ typedef struct {
   unsigned char events;
 
 #define UV_TIMER_PRIVATE_FIELDS                                               \
-  void* heap_node[3];                                                         \
+  union {                                                                     \
+    void* heap_node[3];                                                       \
+    struct uv__queue queue_node;                                              \
+  };                                                                          \
```

Comment:
```suggestion
  } u;                                                                        \
```
Compilers don't always support/accept anonymous unions.